integration-tests/sanity-functional-protractor/page_objects/api-po.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const AbstractPage = require('./abstract-po.js');
+const qs = require('querystring')
+const chakram = require('chakram');
+const port = 8181;
+const getMetadata = '/api/metadata';
+const getArtifact = '/api/artifact';
+
+function RestAPI() {
+
+    this.getMetadata = (queryParams) => {
+        const serviceUrl = `${AbstractPage.getDomain()}:${port}${getMetadata}`;
+        const queryString = qs.stringify(queryParams);
+        const fullUrl = `${serviceUrl}?${queryString}`;
+        return chakram.get(fullUrl, {
+            headers: {
+                'Cache-Control': 'no-cache',
+            }
+        }).then((response) => {
```

Comment:
Not sure if this is needed, when "then" is used in the script itself.

Reply:
done